### DISCUSSION TITLE
Share a problem and fixed about the software (and lib) refusing to start due to version of nvml.dll

### DISCUSSION THREAD

**Discussion by yfdyh000** [LibreHardwareMonitor/LibreHardwareMonitor#1515] [Show and tell]
Recently, I was fiddling with an old graphics card along with NVIDIA drivers. I've noticed that FanControl now stops starting, quietly, no error message. The default configuration or other practices didn't work, and I observe that the issue was also present in LibreHardwareMonitor.
After checking the source code a few times, I see the SystemAccessViolationException error while NvmlInit(), and eventually noticed that this was due to an improper, incompatible newer nvml.dll that had been left in the System32 dir. After removing the file, the issue is gone and it will try to load the file from NVIDIA Corporation\NVSMI dir.

**Comment by uta2000** [comment]
This is a really solid breakdown. You're right that the question universe is small and mostly already covered by the dish modal.

The 4-button approach is more pragmatic:

- **"Why this score?"** — yes, this is the #1 thing I'd want to tap on. We already have the score factors from the content layer, so this is mostly a presentation problem.
- **"What's it like?"** — sensory data is already in the DB, just needs a readable format. Good low-hanging fruit.
- **"Find similar"** — embedding similarity is already built into the scoring engine. Could reuse the same cosine lookup.
- **"Make it work"** — this one's the most interesting and the hardest. Suggesting substitutions requires knowing what's *removable* vs *structural* to a dish (you can skip cilantro garnish, you can't skip the pasta in pasta). Could be a v2 addition.

I'd say MVP is the first three buttons, with "Make it work" as a fast-follow once we figure out the ingredient-removability problem.

One thing I'd add: even with buttons, we might want a lightweight LLM pass for the "Why this score?" explanation. Raw score factors like `tomato: +0.4, cilantro: -0.7` aren't very user-friendly. A single sentence like "You love the roasted tomatoes but cilantro is a dealbreaker for you" is way more useful. That could be a cheap Haiku call with a tight prompt template — no open-ended generation, just score-to-sentence.
